includes/admin/tools/export/class-batch-export-donors.php

```diff
@@ -314,4 +391,15 @@ private function set_donor_data( $i, $data, $donor ) {
 
 	}
 
+	/**
+	 * Unset the properties specific to the donors export.
+	 *
+	 * @param array             $request
+	 * @param Give_Batch_Export $export
+	 */
+	public function unset_properties( $request, $export ) {
```

Comment:
@DevinWalker
Added this function which will run after each step. You can use it to delete any cache or modify any param.